src/include/dhcp-server/commandline.rb

```diff
@@ -124,7 +124,7 @@ def initialize_dhcp_server_commandline(include_target)
               "The hardware type of the host with a fixed address"
             ),
             "type"     => "enum",
-            "typespec" => ["ethernet", "token-ring"]
+            "typespec" => ["ethernet"]
```

Comment:
does this make sense at all? maybe for backward compatibility?

Reply:
I was not sure. I've kept it for compatibility reasons only. AFAIK it is not mandatory anyway.

Reply:
so at least please add comment that it is just for backward compatibility and maybe after string freeze we can add to help text that it is now noop.